recipes/nysdec_state_facility_permits/build.py

```diff
@@ -112,7 +122,7 @@ def _geocode(df: pd.DataFrame) -> pd.DataFrame:
         it = pool.map(geocode, records, 10000)
 
     df = pd.DataFrame(it)
-    df = df[df["geo_grc"] != "71"]
+    # df = df[df["geo_grc"] != GEOSUPPORT_RETURN_CODE_REJECTION]
```

Comment:
Is this supposed to be commented? Just making sure

Reply:
oh yea I can uncomment that. it's having no effect since no records had that return code, but seems like a feature worth still including

unfortunately the [geosupport docs](https://nycplanning.github.io/Geosupport-UPG/chapters/chapterII/section02/) don't seem to detail what that return code would mean:
> A GRC value other than ‘00’ or ‘01’ signifies unsuccessful completion, or rejection, caused by either a system error or a user error.